test/com/minseungkwon/datastructure/Graph/BinaryTreeTest.java

```diff
@@ -267,4 +249,16 @@ public void test_isBalancedUsingObjectFalseCase() {
 
         assertFalse(bt.isBalancedUsingObject());
     }
+
+    @Test
+    public void findNext(){
+        Integer[] arr = makeArrayForTree();
+        BinaryTree<Integer> bt = new BinaryTree<>();
+        bt.makeTree(arr);
+        assertEquals(4, bt.findNext(bt.getRoot().left.right.right).data);
```

Comment:
good, but can we start from the leftmost node and move it to next one by one?
```
Node leftmost = root.something.something;
Node temp = bt.findNext(leftmost);
assert
temp = bf.next...
assert....
```
so far.